### DISCUSSION TITLE
Master Unlocker 1.50.0

### DISCUSSION THREAD

**Discussion by jhowkNx** [jhowkNx/Master-Unlocker#66] [General]
### Character Manager 
   - Added Competitive Stats 
   - Added Races, Darts, Deathmatches, Tennis, Parachuting & Golf
   - Added Total Earned

  ### Acid Lab
   - Added Instant Sell
   - Added Source Assistant

 - Added Unlock Dildodude Camo for Micro SMG & Pump Shotgun
 - Added Bring Gun Van to me (Credits to Rimuwuu)
 - Added Drop RP (based on Meteor Script)
 - Added Give Collectibles (based on Meteor Script)
 - Added Bypass SP Prologue (Credits to Toph Script)
 - Added Enable Snowmen & Trick or Treat collectibles
 - Added Bunker & Hangar Total Earned Stat
 - Improved VIP/CEO/MC - Helper
 - Improved Auto Resupply Warehouse
 - Improved Back to Story Mode

### Code Improvements
   - Improved function that detects a non-GTAO compatible version
   - The script will no longer disable after joining story mode

<hr /><em>This discussion was created from the release <a href='https://github.com/jhowkNx/Master-Unlocker/releases/tag/1.50.0'>Master Unlocker 1.50.0</a>.</em>

**Comment by Mineav1994** [comment]
I downloaded the MasterUnlocker.lua v1.61 version but I can't activate it and it says "Not available in the free version. Get the donor version to enjoy all the features" So is there any way I can do it? enabled to play?

**Comment by Bags101** [comment]
i donated how do i get it to activate donor version?

**Reply by lash7717** [reply]
How much is the donation edition?

**Reply by 524958201** [reply]
I'm the same, how can I activate the donor version
